cms/templates/cms/home_page.html

```diff
@@ -75,7 +75,46 @@
         </div>
       </div>
 
-            <section class="row current-programs-section">
+      <div class="row how-it-works-section">
```

Comment:
It looks to me like this is new content you're adding, and that we were using `<section>` before for this... section. Nonetheless, I've created #1917 to track this.